qemu/tests/cfg/check_block_size.cfg

```diff
@@ -88,3 +88,18 @@
             nic_mode = tap
             physical_block_size_stg = 512
             logical_block_size_stg = 512
+        - 4096_4096_cluster_install:
+            no base
```

Comment:
Is it necessary? In L47, "base" variant only supports 4096_4096. https://github.com/autotest/tp-qemu/blob/c0d8bd805478e4d3568861577dcde6d7d415720c/qemu/tests/cfg/check_block_size.cfg#L46-L47

Reply:
updated